rootfs-builder/suse/config.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<image schemaversion="6.8" name="openSUSE-Leap-Kata" displayname="openSUSE Leap Kata">
+    <description type="system">
+        <author>openSUSE Project</author>
+        <contact>opensuse-factory@opensuse.org</contact>
+        <specification>openSUSE Leap Kata</specification>
+    </description>
+    <preferences>
+        <version>1.0.0</version>
+        <packagemanager>zypper</packagemanager>
+        <locale>en_US</locale>
+        <keytable>us</keytable>
+        <rpm-excludedocs>true</rpm-excludedocs>
+        <type image="vmx" filesystem="ext4" bootloader="grub2" />
+    </preferences>
+    <repository type="rpm-md" alias="Leap_15_0">
+        <source path="obs://openSUSE:Leap:15.0/standard"/>
```

Comment:
Ideally, the two versions in this line and the one above (`15`) would be a parameter (and the file could be `config.xml.in` or something) to make this easy to update.